sql_queries/Create_SpatialSubsetTable.sql

```diff
@@ -0,0 +1,26 @@
+﻿SET SEARCH_PATH = marvl3, public;
+
+DROP TABLE IF EXISTS spatial_subset;
+CREATE TABLE spatial_subset (
+	source_id integer,
+	origin_id text,
+	"LONGITUDE" double precision,
+	"LONGITUDE_QC" text,
+	"LATITUDE" double precision,
+	"LATITUDE_QC" text,
+	"TIME" timestamp with time zone,
+	"TIME_QC" text,
+	"DEPTH" double precision,
```

Comment:
Other parameters than TIME, LATITUDE and LONGITUDE can be stored in single precision.